exercises/acronym/README.md

```diff
@@ -1,5 +1,3 @@
-# Acronym
```

Comment:
Why have all the titles disappeared?

Reply:
The titles disappeared because the original problem-specfications/description.md don't have it in them.

Accumulate README from problem-specfications:
https://github.com/exercism/problem-specifications/blob/master/exercises/accumulate/description.md

Ruby's Accumulate README:
https://github.com/exercism/ruby/blob/master/exercises/accumulate/README.md

I was thinking about adding the titles into the description.md of problem-specifications for every problem so that when generator gets run the titles don't disappear.

Reply:
Please have a read through this thread an the associated issues/PRs:
https://github.com/exercism/problem-specifications/issues/767

Reply:
And: https://github.com/Insti/trackler/blob/master/lib/trackler/implementation.rb#L87
For the code that used to (/still does) generate the readmes.

Reply:
Things are a bit more complicated than normal right now as we're trying to make sure that the exercises are compatible with both the existing exercism.io site and the new https://v2.exercism.io/ (nextercism) site that have slightly different requirements.
It is nextercism that requires that the track have local README.md files.